exporter/sumologicexporter/README.md

```diff
@@ -0,0 +1,49 @@
+# Sumo Logic Exporter
+
+This exporter supports sending logs and metrics data to [Sumo Logic](https://www.sumologic.com/).
+
+The following configuration options are supported:
+
+- `url` (required): Unique URL generated for your HTTP Metrics Source. This is the address to send metrics to.
+- `method` (optional): HTTP method, one of: `POST` or `PUT`. `POST` is used by default if unset.
+- `compress` (optional): Option to enable compression (default false)
+- `compress_encoding` (optional): Compression encoding format, either gzip or deflate (default gzip)
+- `max_request_body_size` (optional): Max HTTP request body size in bytes before compression (if applied). By default 1MB is recommended.
+- `log_format` (optional) (logs only): Format to post logs into Sumo. (default `json`)
```

Comment:
How does this work exactly? when would i use one or the other?

Reply:
It cames from https://github.com/SumoLogic/fluentd-output-sumologic#configuration and AFAIK `text` sends log without metadata and `json` includes additional fields